### PR TITLE
Update libraries to latest nightly

### PR DESCRIPTION
**Pull request by dopefishh**

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -32,7 +35,7 @@ RUN PACKAGES="make subversion ca-certificates gcc git" \
 		*.dcl *.icl \
 		Dockerfile \
 		Makefile \
-	&& rm -rf /opt/clean \
+	&& rm -rf /opt/clean/* \
```
Comment (camilstaps) [inline backend/Dockerfile:38]: Why this change

Comment (dopefishh) [reply backend/Dockerfile:38]: Yes, this is a terrible mistake, this costs us one extra inode!

Comment (camilstaps) [reply backend/Dockerfile:38]: It's not only that, `rm *` takes more CPU cycles because you need to read the directory index; deleting a single directory removes it from the parent in constant time.
